packages/xo-web/src/xo-app/menu/index.js

```diff
@@ -125,6 +154,9 @@ export default class Menu extends Component {
       srs,
       xoaState,
     } = this.props
+    const missingPatchWarningIcon = missingPatchWarning(
+      this.state.missingPatchCount
+    )
```

Comment:
I think that passing the patchCount as a parameter to a function and then return a component is not a really good idea.
Maybe we can delete this function https://github.com/vatesfr/xen-orchestra/pull/4683/files#diff-98351f59c1820b86f4bcd6fdba20515eR34
and add only this here : 
```js
const missingPatchWarningIcon = 
this.state.missingPatchCount > 0 ? <span>
      <Tooltip content={_('homeMissingPatches')}>
        <span className='text-warning'>
          <Icon icon='alarm' />
        </span>
      </Tooltip>
    </span> : null
```